packages/schematics/angular/ng-new/schema.json

```diff
@@ -9,39 +9,39 @@
       "description": "The directory name to create the workspace in."
     },
     "name": {
-      "description": "The name of the workspace.",
+      "description": "The name of the new workspace and initial project.",
       "type": "string",
       "format": "html-selector",
       "$default": {
         "$source": "argv",
         "index": 0
       },
-      "x-prompt": "What name would you like to use for the project?"
+      "x-prompt": "What name would you like to use for the new workspace and initial project?"
     },
     "experimentalIvy": {
-      "description": "EXPERIMENTAL: Specifies whether to create a new application which uses the Ivy rendering engine.",
+      "description": "EXPERIMENTAL: When true, creates a new app that uses the Ivy rendering engine.",
       "type": "boolean",
       "default": false
     },
     "skipInstall": {
-      "description": "Skip installing dependency packages.",
+      "description": "When true, does not install dependency packages.",
       "type": "boolean",
       "default": false
     },
     "linkCli": {
-      "description": "Link CLI to global version (internal development only).",
+      "description": "When true, links CLI to global version (internal development only).",
```

Comment:
Maybe: "When true, links the CLI to the global version (internal development only).",

?? I don't understand how to parse this (global version of what?), but given it's internal maybe that doesn't matter.